### PR TITLE
Create ziue.json

### PR DESCRIPTION
**Pull request by ziuee**

### CODE REVIEW

**Review by hackermondev — CHANGES_REQUESTED** [review]
That's not how you do CNAME with GitHub pages.
https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site

File: domains/ziue.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "description": "Link to my github",
+  "owner": {
+    "username": "ziue",
```
Comment (MaskDuck) [inline domains/ziue.json:4]: rly your gh username?

File: domains/ziue.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+    "discord": "ziue#8495"
```
Comment (MaskDuck) [inline domains/ziue.json:5]: email pls? if not leave email as an empty string

**Review by MaskDuck — CHANGES_REQUESTED** [review]

File: domains/ziue.json (hunk 1 above)
Comment (ziuee) [reply domains/ziue.json:4]: does it gotta be github name?

**Comment by ziuee** [comment]
have a look at new file

File: domains/ziue.json (hunk 1 above)
Comment (MaskDuck) [reply domains/ziue.json]: yes.

**Review by MaskDuck — APPROVED** [review]
sus power uninjected

**Comment by MaskDuck** [comment]
@ziuee 
now go config github pages, see how [here](https://github.com/is-a-dev/register/blob/main/docs/hosted-at/github-pages.md).

**[optional]** star the repo pls

**Pull request merged**